packages/trader/src/Modules/MT5/Containers/mt5-password-modal.jsx

```diff
@@ -236,6 +242,11 @@ const MT5PasswordModal = ({
 }) => {
     const [password, setPassword] = React.useState('');
     const [is_submitting, setIsSubmitting] = React.useState(false); // TODO handle this better
+
+    const is_bvi = React.useMemo(() => {
+        return landing_companies?.mt_financial_company?.financial_stp.shortcode === 'bvi';
```

Comment:
some financial might not offer stp?
```suggestion
        return landing_companies?.mt_financial_company?.financial_stp?.shortcode === 'bvi';
```